proto/kvrpcpb.proto

```diff
@@ -815,6 +815,12 @@ message ScanDetailV2 {
 
     // Total number of bytes from block reads.
     uint64 rocksdb_block_read_byte = 7;
+
+    // Total number of keys from the scan executor (i.e. TableScan or IndexScan) reads.
+    uint64 scan_executor_read_keys = 8;
```

Comment:
how about non executors (like KvScan)?

Reply:
Found that `ScanResponse` doesn't have statistic fields,

https://github.com/pingcap/kvproto/blob/706fcaf286c8dd07ef59349c089f53289a32ce4c/proto/kvrpcpb.proto#L64-L72

is there a plan to add some?